src/config/webpack-shared-config.js

```diff
@@ -61,16 +61,21 @@ module.exports = {
     },
     {
       test: webpackIsomorphicToolsPlugin.regular_expression("styles"),
-      loader: isProduction ? ExtractTextPlugin.extract("style", "css!sass") : "style!css!sass"
+      loader: isProduction ? ExtractTextPlugin.extract({
+        fallbackLoader: "style-loader",
+        loader: [
+          "css-loader",
+          "sass-loader?sourceMap",
+        ],
+      }) : [
+        "style-loader",
+        "css-loader",
+        "sass-loader"
+      ],
     },
-    {
-      test: webpackIsomorphicToolsPlugin.regular_expression("json"),
-      loader: "json"
-    }
```

Comment:
Is this going to break importing json files? or does that work by default in webpack 2?